docs/remote/containers.md

```diff
@@ -384,21 +404,29 @@ A value of `"ui"` instead of `"workspace"` will force the extension to run on th
 
 Containers are isolated environments, so if you want to access a server, service, or other resource inside your container, you will need to either "forward" or "[publish](https://stackoverflow.com/a/22150099)" the port to your host. You can either configure your container to always expose these ports or just forward them temporarily.
 
+### Always forwarding a port
+
+You can specify a list of ports you **always** want to forward when attaching or opening a folder in container by using the `forwardPorts` property in `devcontainer.json`.
+
+```json
+"forwardPorts": [3000, 3001]
+```
+
+Simply reload / reopen the window and the setting will be applied when VS Code connects to the container.
+
 ### Temporarily forwarding a port
 
-Sometimes when developing you may need to access a port in your container that you didn't add to `devcontainer.json`, your Dockerfile, or Docker Compose file. If you want to **temporarily forward** a new port for the duration of the session, run the **Remote-Containers: Forward Port from Container...** command from the Command Palette (`kbstyle(F1)`).
+If you need to access a port that you didn't add to `devcontainer.json` or publish in your Docker Compose fle, you can to **temporarily forward** a new port for the duration of the session, by running the **Remote-Containers: Forward Port from Container...** command from the Command Palette (`kbstyle(F1)`).
```

Comment:
These can be restored by enabling a core setting:
![image](https://user-images.githubusercontent.com/9205389/73637513-cbebe900-4668-11ea-99b7-cf3d0ede8047.png)